pkg/application/application.go

```diff
@@ -71,8 +72,22 @@ func Delete(client *occlient.Client, name string) error {
 
 	labels := applabels.GetLabels(name, false)
 
+	// first delete the services (ServiceInstance in OpenShift terminology)
+	// belonging to the app
+	svcList, err := service.List(client, name)
+	if err != nil {
+		// error is returned when there's no Service Catalog enabled in the service
+		glog.V(4).Infof("Service catalog is not enabled in the cluster, skipping service deletion")
```

Comment:
I'm assuming this based on https://github.com/openshift/odo/blob/974de1fd1285ce8680e7597ba09cf17345e0504d/pkg/odo/cli/service/list.go#L59-L62 in which `List` function is one among more functions which are generalized by this log message